docs/_index.yaml

```diff
@@ -27,18 +27,18 @@ landing_page:
         classname: button
   - classname: devsite-landing-row-cards
     items:
+    - heading: "TensorBoard.dev: a new way to share your ML experiment results"
+      image_path: /resources/images/tf-logo-card-16x9.png
+      path: https://blog.tensorflow.org/2019/12/introducing-tensorboarddev-new-way-to.html
+      buttons:
+      - label: "Read on TensorFlow blog"
+        path: https://blog.tensorflow.org/2019/12/introducing-tensorboarddev-new-way-to.html
     - heading: "The What-If Tool: Code-Free Probing of Machine Learning Models"
```

Comment:
While we are editing this file, perhaps we could also add a link to the Dev Summit talk? https://www.youtube.com/watch?v=xM8sO33x_OU

Reply:
Should we remove another card to make room?